src/main/scala/codacy/tailor/Tailor.scala

```diff
@@ -35,11 +30,12 @@ object Tailor extends Tool {
 
   private lazy val configFileNames = Set(".tailor.yml")
 
-  override def apply(source: Source.Directory,
-                     configuration: Option[List[Pattern.Definition]],
-                     files: Option[Set[Source.File]],
-                     options: Map[Options.Key, Options.Value])(
-      implicit specification: Specification): Try[List[Result]] = {
+  override def apply(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method is longer than 50 lines](https://app.codacy.com/gh/codacy/codacy-tailor/pullRequest?prid=4816796)